### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by MatthewSmithwebdesign**
merge into the master branch

### CODE REVIEW

**Comment by KalobTaulien** [comment]
👍 Looks good to me. Merge when you're happy!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/venv
```
Comment (KalobTaulien) [inline .gitignore]: May need to add this as `venv/`

Also add `*.pyc` to this file

**Review by KalobTaulien — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (MatthewSmithwebdesign) [reply .gitignore]: done and pushed

**Review by KalobTaulien — APPROVED** [review]

**Comment by MatthewSmithwebdesign** [comment]
done and done

**Pull request merged**